public-ui/src/app/data-browser/views/ehr-view/components/concept-row-react.component.tsx

```diff
@@ -0,0 +1,385 @@
+    if (prevProps.selectedConcept !== selectedConcept && selectedConcept.conceptId ===
+    concept.conceptId) {
+        this.setState({
+            showConceptChart: true,
+            graphToShow: domain === 'labs & measurements' ? GraphType.Values : GraphType.BiologicalSex,
+        });
+    }
+    if (selectedConcept && selectedConcept.conceptId !== concept.conceptId &&
+    this.state.showConceptChart) {
+        this.setState({
+            showConceptChart: false
+        });
+    }
+  }
+
+    participantPercentage(count: number) {
+      const {totalParticipants} = this.props;
+      if (!count || count <= 0) { return 0; }
+      let percent: number = count / totalParticipants;
+      percent = percent * 100;
+      return parseFloat(percent.toFixed(2));
+    }
+
+    toggleSynonyms() {
+        this.setState({
+            showMoreSynonyms: !this.state.showMoreSynonyms
+        });
+    }
+
+    toggleDrugBrands() {
+        this.setState({
+            showMoreDrugBrands: !this.state.showMoreDrugBrands
+        });
+    }
+
+    expandRow() {
+        this.setState({
+            showConceptChart: !this.state.showConceptChart,
+            graphToShow: this.props.domain === 'labs & measurements' ? GraphType.Values : GraphType.BiologicalSex
+        });
+    }
+
+    showChart(chartType: string) {
+        const {showConceptChart, graphToShow} = this.state;
+        const {domain} = this.props;
+        if (showConceptChart) {
+            if (chartType === 'sources' && (graphToShow === GraphType.BiologicalSex ||
+            graphToShow === GraphType.Age || graphToShow === GraphType.Values)) {
+                this.setState({
+                    graphToShow: GraphType.Sources
+                });
+            } else if (chartType === 'non-sources' && graphToShow === GraphType.Sources) {
+                this.setState({
+                    graphToShow: GraphType.Values
+                });
+            } else {
+                this.setState({
+                    showConceptChart: !showConceptChart,
+                    graphToShow: chartType === 'sources' ? GraphType.Sources :
+                    (domain === 'labs & measurements' ? GraphType.Values : GraphType.BiologicalSex)
+                });
+            }
+        } else {
+            this.setState({
+                showConceptChart: !showConceptChart,
+                graphToShow: chartType === 'sources' ? GraphType.Sources : (domain === 'labs & measurements'
+                ? GraphType.Values : GraphType.BiologicalSex)
+            });
+        }
+    }
+
+    shareConcept(e: any) {
+      const {concept} = this.props;
+      const el = document.createElement('input');
+      el.value = window.location.origin + window.location.pathname +
+                       '?search=' + concept.conceptId;
+      document.body.appendChild(el);
+      el.select();
+      document.execCommand('copy');
+      document.body.removeChild(el);
```

Comment:
Looks like there's a [javascript ap](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText)i that can do this in a single line. This seemed to work for me:
`navigator.clipboard.writeText(window.location.origin + window.location.pathname + '?search=' + concept.conceptId);`